lib/db.js

```diff
@@ -1311,6 +1327,32 @@ module.exports = (
     })
   }
 
+  function mergeDeviceInfoFromRedis(uid, device, redisSessionTokens, lastAccessTimeEnabled) {
+    // If there's a matching sessionToken in redis, use the more up-to-date
+    // location and access-time info from there rather than from the DB.
+    const token = redisSessionTokens[device.sessionTokenId]
+    const mergedInfo = Object.assign({}, device, token)
+    return {
+      id: device.id,
+      sessionToken: device.sessionTokenId,
```

Comment:
Shouldn't `sessionTokenId` already be in `mergedInfo`? I'm confused why `mergedInfo` is created and then some fields are fetched from `device` instead.

Reply:
I don't think there's any particular reason, it's just that we know some of those keys are only on the `device` that we read from the db.  I'll make them all `mergedInfo` for consistency, assuming that doesn't surprise me with any test bustage.